src/hdmf/term_set.py

```diff
@@ -0,0 +1,97 @@
+from collections import namedtuple
+from .utils import docval
+
+
+class TermSet():
+    """
+    Class for implementing term sets from ontologies and other resources used to define the
+    meaning and/or identify of terms.
+
+    :ivar name: The name of the TermSet
+    :ivar term_schema_path: The LinkML YAML enumeration schema
+    :ivar ontology_version: The version of the ontology
+    :ivar ontology_source_name:  The name of the ontology
+    :ivar ontology_source_uri: The uri of the ontology
```

Comment:
Where are `ontology_version`, `ontology_source_name`, and `ontology_source_uri` define, I didn't see them in the code?

Reply:
Also `name` does not seem to be set in the code.